catapult/utils.py

```diff
@@ -163,7 +163,7 @@ def fatal(message: str, exit_code: int = 1):
 def to_human(data: Any):
     if dataclasses.is_dataclass(data):
         table = [
-            [f.name, getattr(data, f.name)]
+            [f.name, to_human(getattr(data, f.name))]
             for i, f in enumerate(dataclasses.fields(data))
```

Comment:
Not your change, but do we need to use `enumerate` here? We don't seem to be using `i` anywhere.